codec/unknownproto/unknown_fields_test.go

```diff
@@ -29,7 +29,7 @@ func TestRejectUnknownFieldsRepeated(t *testing.T) {
 									F: &testdata.TestVersion2{
 										A: &testdata.TestVersion2{
 											B: &testdata.TestVersion2{
-												H: []*testdata.TestVersion2{
+												H: []*testdata.TestVersion1{
```

Comment:
@odeke-em can you check this? when I ran make proto-gen it forced me to make these changes